.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        java_version:
+          - 11
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java_version }}
+      - name: Build
```

Comment:
Should we use something like `eskatos/gradle-command-action@v1` here? Because on Windows the command `./gradlew` should fail, shouldn't it?

Reply:
It worked with `./gradlew build` as well though I am not sure what kind of magic helped us here :see_no_evil:

Reply:
That's interesting, maybe this is some Microsoft magic to help stupid Linux users, now that GitHub belongs to MS :rofl: